#! /bin/sh /usr/share/dpatch/dpatch-run
## fix-agibin-dir.dpatch by Chris Halls <halls@debian.org>
##
## DP: Fix paths for AGI scripts:
## DP:  * use script path for prefix (not /var/lib/asterisk
## DP:  * logs should be saved in /var/asterisk/log (fixes http://freepbx.org/trac/ticket/1912)

@DPATCH@
diff -urNad freepbx-2.2.1~dfsg~/amp_conf/agi-bin/directory freepbx-2.2.1~dfsg/amp_conf/agi-bin/directory
--- freepbx-2.2.1~dfsg~/amp_conf/agi-bin/directory	2006-08-10 07:38:06.000000000 +0300
+++ freepbx-2.2.1~dfsg/amp_conf/agi-bin/directory	2007-04-26 10:29:30.000000000 +0300
@@ -385,7 +385,7 @@
 
 $agi = new AGI;
 
-if (DEBUG) $logfile = fopen("/tmp/directory.log","w");
+if (DEBUG) $logfile = fopen("/var/log/asterisk/directory.log","w");
 
 $vmconf = array();
 $null = null;
diff -urNad freepbx-2.2.1~dfsg~/amp_conf/agi-bin/fixlocalprefix freepbx-2.2.1~dfsg/amp_conf/agi-bin/fixlocalprefix
--- freepbx-2.2.1~dfsg~/amp_conf/agi-bin/fixlocalprefix	2006-12-06 22:50:30.000000000 +0200
+++ freepbx-2.2.1~dfsg/amp_conf/agi-bin/fixlocalprefix	2007-04-26 10:29:30.000000000 +0300
@@ -69,7 +69,7 @@
 
 */
 
-define("AGIBIN_DIR", "/var/lib/asterisk/agi-bin");
+define("AGIBIN_DIR", dirname(__FILE__));
 define("LOCALPREFIX_FILE", "/etc/asterisk/localprefixes.conf");
 
 include(AGIBIN_DIR."/phpagi.php");
diff -urNad freepbx-2.2.1~dfsg~/amp_conf/agi-bin/recordingcheck freepbx-2.2.1~dfsg/amp_conf/agi-bin/recordingcheck
--- freepbx-2.2.1~dfsg~/amp_conf/agi-bin/recordingcheck	2006-06-26 02:46:50.000000000 +0300
+++ freepbx-2.2.1~dfsg/amp_conf/agi-bin/recordingcheck	2007-04-26 10:29:30.000000000 +0300
@@ -17,7 +17,7 @@
 The program checks determines if Asterisk should record a call
 */
 
-define("AGIBIN_DIR", "/var/lib/asterisk/agi-bin");
+define("AGIBIN_DIR", dirname(__FILE__));
 
 include(AGIBIN_DIR."/phpagi.php");
 
diff -urNad freepbx-2.2.1~dfsg~/amp_conf/bin/archive_recordings freepbx-2.2.1~dfsg/amp_conf/bin/archive_recordings
--- freepbx-2.2.1~dfsg~/amp_conf/bin/archive_recordings	2006-06-26 02:46:50.000000000 +0300
+++ freepbx-2.2.1~dfsg/amp_conf/bin/archive_recordings	2007-04-26 10:32:05.000000000 +0300
@@ -2,7 +2,7 @@
 <?php
 
 define("AMP_CONF", "/etc/amportal.conf");
-define("PHP_ASMANAGER", "/var/lib/asterisk/bin/php-asmanager.php");
+define("PHP_ASMANAGER", "/usr/share/freepbx/common/php-asmanager.php");
 define("MONITOR_DIR", "/var/spool/asterisk/monitor");
 
 function out($text) {
